core/iwasm/libraries/libc-builtin/libc_builtin_wrapper.c

```diff
@@ -16,7 +16,7 @@
 void
 wasm_runtime_set_exception(wasm_module_inst_t module, const char *exception);
 
-uint32
+uint64
```

Comment:
Could you also modify the realloc_wrapper function in this file?
How about changing to somewhat like below:
```C
static uint32
realloc_wrapper(wasm_exec_env_t exec_env, uint32 ptr, uint32 new_size)
{
    wasm_module_inst_t module_inst = get_module_inst(exec_env);
    uint64 ret_offset;

    ret_offset = wasm_runtime_module_realloc(module_inst, ptr, new_size, NULL);
    bh_assert(ret_offset < UINT32_MAX);
    return (uint32)ret_offset;
}
```